package.json

```diff
@@ -27,7 +28,9 @@
     "fs-jetpack": "^2.2.3",
     "graphql": "^14.5.8",
     "ink": "^2.5.0",
+    "lodash": "^4.17.15",
```

Comment:
Forgot to mention, about this. We need to get to a tree shaking point and then it will be fine. Or we can just find a deep merge standalone package. I tried `lodash-es` and got errors with jest. But anyways, don't think it would have helped yet, as we're not bundling the user's app.

![image](https://user-images.githubusercontent.com/284476/72038413-49415b00-326f-11ea-96ca-35163ff8ee05.png)